docs/api.yml

```diff
@@ -693,14 +734,19 @@ components:
         day:
           type: integer
           example: 20
+        season:
+          type: string
+          example: "Spring"
+          description: "Season of publication (for example, Spring, Third Quarter)"
     NoteHistoryDate:
       type: object
       properties:
         date_type:
           type: string
         date:
           type: string
-          format: date
+          example: "'2020-03-01' or '2020-11' or '2021'"
```

Comment:
Update per suggest change. Looks like this:

![image](https://user-images.githubusercontent.com/57392136/76410171-68e82300-6387-11ea-908c-5d6568d7e02e.png)

![image](https://user-images.githubusercontent.com/57392136/76410229-7ef5e380-6387-11ea-8724-f9e9e9d1b515.png)